source/Generic/PlayState/Localization/en_US.xaml

```diff
@@ -41,7 +45,8 @@ For better compatibility with games, some adjustments to your Windows configurat
     <sys:String x:Key="LOCPlayState_SettingGlobalOnlySuspendPlaytimeLabel">Only suspend playtime count globally and not the game processes</sys:String>
     <sys:String x:Key="LOCPlayState_SettingGlobalOnlySuspendPlaytimeTooltip" xml:space="preserve">By enabling this option, all games won't be suspended when using the configured hotkey and only the time will be counted to substracted from the game playtime when it stops.
     
-This can also be applied individually per game by adding a feature named "[PlayState] Suspend playtime only" to a game</sys:String>
+The state of this setting can be overridden individually per game by adding a feature named "[PlayState] Override suspend playtime only setting" to a game.
```

Comment:
I think it's confusing how it would work. Instead of opposites, it should say what it does in each mode. As examples, this is done in the Special K Helper and Steam Launcher Utility extensions 
![image](https://user-images.githubusercontent.com/1389286/160935036-80245af4-0ade-4cea-958a-ebcdda849847.png)

Reply:
So let the previous feature as it is and just create a new one with the other result, right?

Reply:
Yes, I think it can get very confusing otherwise. So the solution is to have two features: `[PlayState] Suspend Playtime` and `[PlayState] Suspend Processes`. This would override the current PlayState mode setting and would have the benefit of being very clear how the extension will behave with the specific game